packages/cloudflare/src/handler.ts

```diff
@@ -40,5 +54,52 @@ export function withSentry<E extends ExportedHandler<any>>(
     (handler.fetch as any).__SENTRY_INSTRUMENTED__ = true;
   }
 
+  if (
+    'scheduled' in handler &&
+    typeof handler.scheduled === 'function' &&
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+    !(handler.scheduled as any).__SENTRY_INSTRUMENTED__
+  ) {
+    handler.scheduled = new Proxy(handler.scheduled, {
+      apply(target, thisArg, args: Parameters<ExportedHandlerScheduledHandler<ExtractEnv<E>>>) {
+        const [event, env, context] = args;
+        return withIsolationScope(isolationScope => {
+          const options = optionsCallback(env);
+          const client = init(options);
+          isolationScope.setClient(client);
+
+          addCloudResourceContext(isolationScope);
+
+          return startSpan(
+            {
+              op: 'faas.cron',
+              name: `Scheduled Cron ${event.cron}`,
+              attributes: {
+                'faas.cron': event.cron,
+                'faas.time': new Date(event.scheduledTime).toISOString(),
+                'faas.trigger': 'timer',
+                [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.faas.cloudflare',
+                [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'task',
+              },
+            },
+            async () => {
+              try {
+                return await (target.apply(thisArg, args) as ReturnType<typeof target>);
+              } catch (e) {
+                captureException(e, { mechanism: { handled: false, type: 'cloudflare' } });
+                throw e;
+              } finally {
+                context.waitUntil(flush(2000));
```

Comment:
`waitUntil` will execute keep the serverless function alive until we flush out all sentry requests, but it will ensure that this happens after a response is sent. Therefore flushing to sentry does not block sending a response back to whatever sent the request to the cloudflare worker.